### PR TITLE
Add service spec

### LINKED ISSUE #2: Add swagger
**Is your feature request related to a problem? Please describe.**
There is no swagger to show api spec of the service

**Describe the solution you'd like**
create openapii3.yaml with all the service spec

### PR DESCRIPTION
**Pull request by galta95**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Documentation: added openapi3 yaml for api spec   | ✔                                                                       |

Related issues: #2
Closes #2

Further  information:
The spec includes `/converter` endpoint for converting external data to geoJson format.
geoJson schema for response body was taken from  [geojson/schema](https://github.com/geojson/schema)

### CODE REVIEW

File: openapi3.yaml [hunk 1]
```diff
@@ -49,17 +42,326 @@ components:
...
-          format: int64
-        name:
-          type: string
-        description:
+        type:
           type: string
+          enum:
+            - Feature
+        properties:
+          type: object
```
Comment (syncush) [inline openapi3.yaml:57]: please add an example of a free form object with some properties and values

File: openapi3.yaml [hunk 2]
```diff
@@ -1,41 +1,34 @@
...
+    post:
       tags:
-        - resourceName
-      summary: gets the resource
+        - convert
+      summary: convert body record to geoJson
       responses:
         200:
-          description: OK
+          description: converted
```
Comment (syncush) [inline openapi3.yaml:19]: I think the URL should be POST `/sources/{external_source_name}/convert`

even though we are currently working on one external source, there is no stopping us from generalizing the service for more scehmas

File: commitlint.config.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 module.exports = {
     extends: ['@commitlint/config-conventional'],
     rules: {
-        "scope-enum": [2, "always", ["deps", "configurations"]],
+        "scope-enum": [2, "always", ["deps", "configurations", "api", "convert"]],
```
Comment (syncush) [inline commitlint.config.js:4]: `api` is not a good scope

if you accept that we need to change the API to be by `source/{source_name}` then a good scope would be `external-sources`

**Review by syncush — CHANGES_REQUESTED** [review]

File: openapi3.yaml (hunk 1 above)
Comment (galta95) [reply openapi3.yaml:79]: Done

File: openapi3.yaml (hunk 2 above)
Comment (galta95) [reply openapi3.yaml]: Done

File: commitlint.config.js (hunk 3 above)
Comment (galta95) [reply commitlint.config.js]: Done

**Review by syncush — APPROVED** [review]

**Pull request merged**